_code-samples/nft-modular-tutorials/account-support.js

```diff
@@ -0,0 +1,207 @@
+// ******************************************************
+// ************* Get the Preferred Network **************
+// ******************************************************  
+
+function getNet() {
+  let net
+  if (document.getElementById("tn").checked) net = "wss://s.altnet.rippletest.net:51233/"
+  if (document.getElementById("dn").checked) net = "wss://s.devnet.rippletest.net:51233/"
+  return net
+} // End of getNet()
+
+// *******************************************************
+// ************* Get Account *****************************
+// *******************************************************
+
+async function getAccount() {
+  let net = getNet()
+  const client = new xrpl.Client(net)
+  await client.connect()
+  resultField.value = `===Getting Account===\n\nConnected to ${net}.`
+  try {
+    let faucetHost = null
+    const my_wallet = (await client.fundWallet(null, { faucetHost})).wallet
+    const newAccount = [my_wallet.address, my_wallet.seed]
+    return (newAccount)
+  }
+  catch (error) {
+    console.error('===Error getting account:', error);
+    results += `\nError: ${error.message}\n`
+    resultField.value = results
+    throw error; // Re-throw the error to be handled by the caller
+  }
+  finally {
+    // Disconnect from the client
+    await client.disconnect();
```

Comment:
Probably good to add this here, just like we do in **_code-samples/modular-tutorials/account-support.js**

```suggestion
    if (client && client.isConnected()) {
      await client.disconnect();
    }
```